docs/Developers/Development_Setup.rst

```diff
@@ -17,48 +17,57 @@ Here we will be showing how to have a virtualenv using :code:`venv` and
 of the standard library and requires no extra installation. However,
 you can use whichever you find suitable to yourself.
 
-Using venv
-~~~~~~~~~~
+Using virtualenv
```

Comment:
Why did you remove the section on using `venv`? Please restore it. `venv` is not the same as `virtualenv`